google/dm-solutions/README.md

```diff
@@ -0,0 +1,6 @@
+# Deployment Manager Solutions
```

Comment:
Should thus directory be named just solutions, since it is under deployment manager's repo?